server/app/dal/Dao.scala

```diff
@@ -62,6 +72,11 @@ class DaoImpl @Inject()(cluster: CassandraCluster)(implicit ec: ExecutionContext
       val limitInt: Integer = limit
       session.executeAsync(getLastTasksStatement.bind(year(yearAgo), `type`.toString, limitInt))
     }.map(allTasks))
+
+  def getTask(year: Long, `type`: TaskType, timeuuid: UUID): Future[Option[Task]] = TryFuture(
+    toFuture {
+      session.executeAsync(getTaskStatement.bind(new Timestamp(year), `type`.toString, timeuuid))
```

Comment:
Dao shouldn't do any tricks like `new Timestamp(year)`, I propose something like this https://github.com/DmytroOrlov/devgym/commit/acda8384b17c5d0c1066227f5aaf055b9552ac02